test/scenario/broker/src/test/test.js

```diff
@@ -111,29 +98,46 @@ const complexMeteringPlan = {
   ]
 };
 
-describe('Abacus Broker Acceptance test', function() {
-  this.timeout(totalTimeout);
+describe('Abacus Broker Scenario test', function() {
+  this.timeout(testEnv.totalTimeout);
 
   let app;
   let orgId;
   let spaceId;
   let testAppClient;
 
+  let App;
+  let Service;
+  let abacusClient;
+
   before(() => {
+    checkCorrectSetup(testEnv);
+
+    const testUtils = cfUtil({
+      api: testEnv.api,
+      user: testEnv.user,
+      password: testEnv.password
+    });
+
+    App = testUtils.App;
+    Service = testUtils.Service;
+
+    abacusClient = abacusUtil(testEnv.provisioningUrl, testEnv.collectorUrl, testEnv.reportingUrl);
+
     app = App.deploy({
       target:{
         orgName: testEnv.org,
         spaceName: testEnv.space
       },
       app: {
         name: `${moment.utc().valueOf()}-test-app`,
-        manifest: `${__dirname}/apps/test-app/manifest.yml`
+        manifest: `${getAbacusDirPath()}/test/test-utils/apps/test-app/manifest.yml`
```

Comment:
I am not sure about that "hidden" dependency between modules. I'd expect that the test would get the needed resources though its dependencies and not by traversing the directory tree.